id: PYSEC-0000-CVE-2009-2701
package:
  name: zodb3
  ecosystem: PyPI
details: Unspecified vulnerability in the Zope Enterprise Objects (ZEO) storage-server
  functionality in Zope Object Database (ZODB) 3.8 before 3.8.3 and 3.9.x before 3.9.0c2,
  when certain ZEO database sharing and blob support are enabled, allows remote authenticated
  users to read or delete arbitrary files via unknown vectors.
affects:
  ranges:
  - type: ECOSYSTEM
    introduced: "3.8"
    fixed: 3.8.3
  - type: ECOSYSTEM
    introduced: "3.9"
    fixed: 3.9.0c2
references:
- type: WEB
  url: https://mail.zope.org/pipermail/zope-announce/2009-September/002221.html
- type: WEB
  url: http://pypi.python.org/pypi/ZODB3/3.9.0c2
- type: WEB
  url: http://pypi.python.org/pypi/ZODB3/3.8.3
- type: WEB
  url: http://www.vupen.com/english/advisories/2009/2534
aliases:
- CVE-2009-2701
modified: "2009-09-09T04:00:00Z"
published: "2009-09-08T18:30:00Z"
